livechat/webhooks/v35.py

```diff
@@ -0,0 +1,370 @@
+    chat_id: str
+    thread_id: str
+    properties: dict
+
+
+@dataclass
+class EventPropertiesUpdated:
+    chat_id: str
+    thread_id: str
+    event_id: str
+    properties: dict
+
+
+@dataclass
+class EventPropertiesDeleted:
+    chat_id: str
+    thread_id: str
+    event_id: str
+    properties: dict
+
+
+# Thread tags
+
+
+@dataclass
+class ThreadTagged:
+    chat_id: str
+    thread_id: str
+    tag: str
+
+
+@dataclass
+class ThreadUntagged:
+    chat_id: str
+    thread_id: str
+    tag: str
+
+
+# Status
+
+
+@dataclass
+class RoutingStatusSet:
+    agent_id: str
+    status: str
+
+
+# Customers
+
+
+@dataclass
+class IncomingCustomer:
+    customer: dict
+
+
+@dataclass
+class CustomerSessionFieldsUpdated:
+    id: str
+    session_fields: list
+    active_chat: dict = None
+
+
+# Configuration
+
+
+@dataclass
+class AgentCreated:
+    id: str
+    name: str
+    awaiting_approval: bool
+    role: str = None
+    avatar: str = None
+    job_title: str = None
+    mobile: str = None
+    max_chats_count: int = None
+    groups: list = None
+    notifications: list = None
+    email_subscriptions: list = None
+    work_scheduler: dict = None
+
+
+@dataclass
+class AgentApproved:
+    id: str
+
+
+@dataclass
+class AgentUpdated:
+    id: str
+    name: str = None
+    role: str = None
+    avatar: str = None
+    job_title: str = None
+    mobile: str = None
+    max_chats_count: int = None
+    groups: list = None
+    notifications: list = None
+    email_subscriptions: list = None
+    work_scheduler: dict = None
+
+
+@dataclass
+class AgentSuspended:
+    id: str
+
+
+@dataclass
+class AgentUnsuspended:
+    id: str
+
+
+@dataclass
+class AgentDeleted:
+    id: str
+
+
+@dataclass
+class AutoAccessAdded:
+    id: str
+    description: str
+    access: dict
+    conditions: dict
+    next_id: str = None
+
+
+@dataclass
+class AutoAccessUpdated:
+    id: str
+    description: str = None
+    access: dict = None
+    conditions: dict = None
+    next_id: str = None
+
+
+@dataclass
+class AutoAccessDeleted:
+    id: str
+
+
+@dataclass
+class BotCreated:
+    id: str
+    name: str
+    default_group_priority: str
+    owner_client_id: str
+    avatar: str = None
+    max_chats_count: int = None
+    groups: list = None
+    work_scheduler: dict = None
+    timezone: str = None
+    job_title: str = None
+
+
+@dataclass
+class BotUpdated:
+    id: str
+    name: str = None
+    avatar: str = None
+    max_chats_count: int = None
+    default_group_priority: str = None
+    groups: list = None
+    work_scheduler: dict = None
+    timezone: str = None
+    job_title: str = None
+
+
+@dataclass
+class BotDeleted:
+    id: str
+
+
+@dataclass
+class GroupCreated:
+    id: int
+    name: str
+    language_code: str
+    agent_priorities: dict
+
+
+@dataclass
+class GroupDeleted:
+    id: str
+
+
+@dataclass
+class GroupUpdated:
+    id: int
+    name: str = None
+    language_code: str = None
+    agent_priorities: dict = None
+
```

Comment:
TagCreated, TagDeleted, and TagUpdated are missing (according to https://developers.livechat.com/docs/management/webhooks/v3.5)

Reply:
https://github.com/livechat/lc-sdk-python/pull/78/commits/a9adbf0abc44003d09a0ca99eb4458d8c9ce7ea7